naga/src/ir/mod.rs

```diff
@@ -386,10 +396,10 @@ pub enum BuiltIn {
     FragDepth,
     PointCoord,
     FrontFacing,
-    PrimitiveIndex,
+    PrimitiveIndex, // Also for mesh output
```

Comment:
suggestion: Shall we turn this and the other small comments around it into doc commenta instead? They seem like helpful notes in a maintainer's or dev.'s experience, not just somebody who happens to be looking at this code.